spec/mailers/notify_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+RSpec.describe Notify, type: :mailer do
+  let!(:report) { create(:monthly_report, :with_comments, :with_tags) }
+  let(:user) { create(:user) }
+  let(:from) { "システム管理者 "+"#{Settings.mailer[:from]}" } 
+
+  after(:all) do
+    ActionMailer::Base.deliveries.clear
+  end
+
+  describe "monthly_report_registration" do
+    let(:mail) { Notify.monthly_report_registration(user.id,report.id) }
+    let(:name) { user.name }
+    let(:created_at) { report.target_month.strftime('%Y年%m月') }
+    let(:title) { "#{name}が#{created_at}の月報を登録しました" }
+    
+    it "deliver_now" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)